runtime/src/runtime/mod.rs

```diff
@@ -24,15 +24,15 @@ pub use self::randomness::DomainSeparationTag;
 use crate::runtime::builtins::Type;
 use crate::ActorError;
 
+mod actor_code;
+pub mod builtins;
+pub mod policy;
+mod randomness;
+
 #[cfg(feature = "fil-actor")]
 mod actor_blockstore;
 #[cfg(feature = "fil-actor")]
-pub mod builtins;
```

Comment:
The fil-actors line should have gone one line below this (over `pub mod fvm`, not `pub mod builtins`).